recipes/slidingwindow/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "slidingwindow" %}
+{% set version = "0.0.14" %}
+{% set sha256 = "4c52adf24e2dd6786b9c1c49144083254d9bd9de655b4f2f17bd78cc41a01e49" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: {{ sha256 }}
```

No need to use a variable as it's only used in one place:
```suggestion
  sha256: 4c52adf24e2dd6786b9c1c49144083254d9bd9de655b4f2f17bd78cc41a01e49
```